src/test/java/org/jenkinsci/plugins/github/config/ConfigAsCodeTest.java

```diff
@@ -0,0 +1,64 @@
+package org.jenkinsci.plugins.github.config;
+
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.jenkinsci.plugins.github.test.GitHubServerConfigMatcher.withClientCacheSize;
+import static org.jenkinsci.plugins.github.test.GitHubServerConfigMatcher.withCredsId;
+import static org.jenkinsci.plugins.github.test.GitHubServerConfigMatcher.withApiUrl;
+import static org.jenkinsci.plugins.github.test.GitHubServerConfigMatcher.withIsManageHooks;
+import static org.jenkinsci.plugins.github.test.GitHubServerConfigMatcher.withName;
+
+public class ConfigAsCodeTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void shouldSupportConfigurationAsCode() throws Exception {
+        ConfigurationAsCode.get().configure(
+                ConfigAsCodeTest.class.getResource("configuration-as-code.yml").toString());
```

Comment:
once you add the `tests` classifier, please use the annotation `@ConfiguredWithCode("configuration-as-code.yml")`